packages/dataviews/src/stories/dataform.story.tsx

```diff
@@ -1575,6 +1575,7 @@ const LayoutCardComponent = ( {
 
 	const form: Form = useMemo(
 		() => ( {
+			layout: { type: 'card' },
```

Comment:
🤔 This should not be necessary because every field declares its layout, and all the 3 top-level fields that create a card have `card` as the layout. That should be enough to match any styles. I'll take a look.

Reply:
Thank you. 

In case it's not clear, this is about the spacing between the cards themselves (which should be 24px), not the fields _within_ the cards (which should be 16px).

Reply:
I've approved this PR, but we can continue the conversation.

I thought about this, but I've noticed we mix panel and regular items in a few places (QuickEdit, for example). If the field themselves provide the spacing, and each layout has its own, wouldn't this create different spacing between fields in the same form?